src/components/render-page/word-counter.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import wordsCount from 'words-count'
+
+const EXCLUDED_TAGS = ['img', 'pre', 'nav']
+
+export const countWordsRecursive = (node: Element | ChildNode): number => {
```

Comment:
Is it important for a caller to know that this method works recursivly? I don't see whats wrong with just "countWords".

Reply:
I'd say it's better for code understanding if someone can quickly determine that this method is called recursively.

Reply:
The name is more important for the person that users the function and not the person that is reviewing it. The name should describe the purpose of the function and not how it works.

Reply:
changed if you insist on it